config/manager/manager.yaml

```diff
@@ -44,7 +44,6 @@ spec:
       - command:
         - /manager
         args:
-        - --leader-elect
         - --cert-dir=/tmp
         - --fluent-bit-cpu-limit=1
         - --fluent-bit-memory-limit=1Gi
```

Comment:
`fluent-bit-cpu-limit`,  `fluent-bit-memory-limit`, `fluent-bit-denied-output-plugins`, `fluent-bit-max-pipelines` and all trace-collector flags could also be removed as we use the default value.